opendevin/runtime/utils/bash.py

```diff
@@ -0,0 +1,87 @@
+def split_bash_commands(commands):
```

Comment:
QQ: Wondering wether this function will be usefull in our event stream client when execute cmd.

Reply:
Yep - we'll need to use it when we want to handle the test case where multiple commands are issued at the same time.